package.json

```diff
@@ -109,6 +109,7 @@
     "typescript": "^4.3.2"
   },
   "resolutions": {
-    "babel-loader": "8.1.0"
+    "babel-loader": "8.1.0",
+    "@storybook/react/**/loader-utils": "^2.0.4"
```

Comment:
was there no `@storybook/react` version available that upgraded this enough? This is acceptable, but just curious.

Reply:
oh yeah, i went down that path and it was opening up pandora's box. nothing would run and a ton of errors.
we may need to rehaul a lot of our dev dependencies at some point.

Reply:
ouch 👍